Cargo.toml

```diff
@@ -113,7 +113,7 @@ redis = { version = "0.21", features = [
   "tokio-comp",
   "connection-manager",
 ], optional = true }
-reqsign = "0.4"
+reqsign = "0.4.3" # must above 0.4.3 to use new api
```

Comment:
No, just remove your local `Cargo.lock` files.

Reply:
I don't really get it...